ajax/libs/vertx/package.json

```diff
@@ -17,6 +17,18 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/vert-x/vert.x"
+    "url": "git://github.com/vert-x3/vertx-bus-bower"
```

Comment:
@clairetsai818 the development git repo lives here: `git://github.com/vert-x3/vertx-web` and the source file `vertx-web/vertx-web/src/client/vertx-eventbus.js`.